carfield.mk

```diff
@@ -316,7 +316,7 @@ pulpd-sw-init: $(PULPD_ROOT) $(PULPD_ROOT)/pulp-runtime $(PULPD_ROOT)/regression
 $(PULPD_ROOT)/pulp-runtime: $(PULPD_ROOT)
 	$(MAKE) -C $(PULPD_ROOT) pulp-runtime
 $(PULPD_ROOT)/regression-tests: $(PULPD_ROOT)
-	$(MAKE) -C $(PULPD_ROOT) regression-tests
+	git clone git@github.com:pulp-platform/regression_tests.git -b lv/car-wce $@
```

Comment:
I would not change it this way. I saw that in the [regression tests](https://github.com/pulp-platform/regression_tests/commits/lv/car-wce/) you have added just a couple of commits on top of the branch that was natively used by the [pulp_cluster](https://github.com/pulp-platform/pulp_cluster/blob/yt/rapidrecovery/Makefile#L83), thus I would suggest you open a PR in the regression tests so that we do not have to change the fragmanted makefile here.

Reply:
I agree with @yvantor

Reply:
The PR was merged and the changes reverted :smile: